build-support/bin/generate_github_workflows.py

```diff
@@ -305,9 +306,30 @@ def generate() -> dict[Path, str]:
             },
         }
     )
+
+    audit_yaml = yaml.dump(
+        {
+            "name": "Cargo Audit",
+            # 11:11pm
```

Comment:
It would be good to add a comment explaining why this time was chosen, even if it's just "an arbitrary time late at night every day"